apps/desktop/src/components/editor-area/index.tsx

```
@@ -433,10 +452,18 @@ export function useEnhanceMutation({
+
       for await (const chunk of fullStream) {
         if (chunk.type === "text-delta") {
           acc += chunk.textDelta;
         }
+        if (chunk.type === "error") {
+          if (originalContentRef.current !== "" && acc === "") {
+            setEnhancedContent(originalContentRef.current);
+          }
+          throw new Error("Error occured right away");
+          break;
+        }
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)